src/analytics/customEventTracker.ts

```diff
@@ -0,0 +1,13 @@
+import { TrackingEventName } from "./constants";
+
+export function trackCustomEvent(
+  eventName: TrackingEventName,
+  eventParameters: Record<string, any> = {}
+) {
+  console.log("Custom Event:", eventName);
+  console.log("Event Parameters:", eventParameters);
+
+  if (window.gtag) {
```

Comment:
Short cut to this and no need the if?

window.gtag?.("event", eventName, eventParameters);